src/modv/draw.js

```diff
@@ -1,18 +1,61 @@
 import { modV } from '@/modv';
-import render2d from '@/modv/renderers/2d';
-import { render as renderShader } from '@/modv/renderers/shader';
-import { render as renderIsf } from '@/modv/renderers/isf';
+// import render2d from '@/modv/renderers/2d';
+// import { render as renderShader } from '@/modv/renderers/shader';
+// import { render as renderIsf } from '@/modv/renderers/isf';
```

Comment:
Please remove unused imports.

Reply:
@TimPietrusky This should be resolved now